esm/streams/application-logs.js

```diff
@@ -22,20 +28,13 @@ export class ApplicationLogStream extends CleverCloudSse {
    * @param {string} options.field[]
    * @param {number} options.throttleElements
    * @param {number} options.throttlePerInMilliseconds
-   * @param {object} options.retryConfiguration
-   * @param {boolean} options.retryConfiguration.enabled
-   * @param {number} options.retryConfiguration.backoffFactor
-   * @param {number} options.retryConfiguration.initRetryTimeout
-   * @param {number} options.retryConfiguration.maxRetryCount
    *
    */
-  constructor ({ apiHost, tokens, ownerId, appId, ...options }) {
-    super(apiHost, tokens, options.retryConfiguration || {});
-
+  constructor ({ apiHost, tokens, ownerId, appId, retryConfiguration, ...options }) {
+    super(apiHost, tokens, retryConfiguration ?? {});
     this.ownerId = ownerId;
     this.appId = appId;
-    // retryConfiguration key is used for another thing (see upper)
-    this.options = { ...options, retryConfiguration: undefined};
+    this.options = options;
```

Comment:
with it, the retryConfiguration parameters will be merged with options like limit or since, which will be added as query parameters

Reply:
I changed the constructor to this so `retryConfiguration` is not in the options:

```
constructor ({ apiHost, tokens, ownerId, appId, retryConfiguration, ...options }) {
```